recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
Maybe we could update how `conda skeleton pypi` works based on this. Not sure if we want to. Just something to think about.